app/views/groups/show.html.haml

```diff
@@ -119,7 +120,18 @@
             %i.ofn-i_050-mail-circle
           =link_to_service "http://", @group.website, title: t(:groups_contact_website) do
             %i.ofn-i_049-web
-        %p
-          &nbsp;
+          .powered-by-embedded
+            %img{src: '/favicon.ico'}
+            %span
+              = t 'powered_by'
+            %span
+              = t 'title'
 
 = render "shared/footer"
+
+:javascript 
```

Comment:
I'm not sure about this. We usually don't include random javascript on pages. Can we use `ng-attr` like above or include this in the already existing Angular controller?

Reply:
I've now implemented an angular based solution for this. Can't use ng-attr as far as I can see because we need to find all child elements of the about div and add target = "_blank" to each of them. I don't know a way to do this with ng-attr. I've written a directive for it instead though and this seems to work well. What do you think?